mmv1/third_party/terraform/go.mod.erb

```diff
@@ -24,8 +24,8 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/mitchellh/hashstructure v1.1.0
 	github.com/sirupsen/logrus v1.8.1
-	golang.org/x/net v0.15.0
-	golang.org/x/oauth2 v0.12.0
+	golang.org/x/net v0.16.0
```

Comment:
Any particular reason to update libraries other than oauth2?

Reply:
The net requirement got bumped from 0.15.0 -> 0.16.0 in the most recent release of oauth2

Reply:
I guess same reason for sys, crypto and term right?

Reply:
Yep, the requirement for all those packages got bumped with the 15->16 version change for net